openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx

```diff
@@ -736,7 +738,7 @@ const PipelineDetails = ({
 
   useEffect(() => {
     getEntityFeedCount();
-  }, [pipelineFQN, description, pipelineDetails]);
+  }, [pipelineFQN]);
```

Comment:
as we are not using pipelineFQN in useEffect, we can remove it. let me know if i am missing something

Reply:
we are getting the `pipelineFQN` by props, and we are expecting it to call the useEffect on pipelineFQN change.